### ISSUE TITLE
Saito website wallet settings menu not working

### ISSUE THREAD

**Issue by Kromtastic** [SaitoTech/saito-lite-rust#741]
Tested on several browsers and wallets. 

Steps: 

1) Use the Burger Menu on the Arcade

2) Click on Settings

![image](https://user-images.githubusercontent.com/98101498/178151795-3b66ab4a-e0b1-43d3-9dd2-3eb1a78e8cdd.png)

3) Click on Settings, nothing happens.

![image](https://user-images.githubusercontent.com/98101498/178151815-3c8a9911-8d13-4f48-9036-32ee5dd132fe.png)

Without the Settings menu, users won't be able to Restore/Save/Import their wallets, apply Nicknames, etc.

**Comment by The1Duke** [comment]
@f8daniel this issue is still present on the after the latest update.

**Comment by f8daniel** [comment]
Yeah, I haven't been working on it. I thought Dave said somewhere in a
group chat that he fixed it

**Comment by karlsaito** [comment]
error is both present in staging and prod:

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/96273856/181159419-e512f406-9670-4968-9ab3-f5eca2a174f5.png">
